spec/lib/champaign_queue/clients/sqs_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+describe ChampaignQueue::Clients::Sqs do
+  context "with SQS_QUEUE_URL" do
+    it "delivers payload to AWS SQS Queue" do
+
+      expected_arguments = {
+        queue_url: "http://example.com",
+        message_body: {foo: :bar}.to_json
+      }
+
+      expect_any_instance_of(Aws::SQS::Client).to(
+        receive(:send_message).with( expected_arguments )
+      )
+
+      ChampaignQueue::Clients::Sqs.push({foo: :bar})
```

Comment:
Are we expecting other clients than the SQS client for the ChampaignQueue namespace?